### PR TITLE
Fixing #126 redirecting after draft delete

### PR DESCRIPTION
**Pull request by ahmgeek**

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+ruby-2.0.0-p594
```
Comment (mkhatib) [inline .ruby-version]: Drop this file we already have one under backend/ (which where it is needed)

File: .ruby-gemset [hunk 2]
```diff
@@ -0,0 +1 @@
+manshar
```
Comment (mkhatib) [inline .ruby-gemset]: Drop?

File: web-client/app/scripts/controllers/profileController.js [hunk 3]
```diff
@@ -29,14 +28,18 @@ angular.module('webClientApp')
     };
 
     $scope.editProfile = function () {
-      $location.path('/profiles/' + $rootScope.currentUser.id + '/edit');
+      $location.path('/profiles/' + $rootScope.currentuser.id + '/edit');
```
Comment (mkhatib) [inline web-client/app/scripts/controllers/profileController.js]: currentUser not currentuser

File: web-client/app/scripts/controllers/profileController.js [hunk 4]
```diff
@@ -29,14 +28,18 @@ angular.module('webClientApp')
...
     };
 
     var deleteSuccess = function () {
       $analytics.eventTrack('Article Deleted', {
         category: 'Article'
       });
-      $location.path('/');
+
+      //don't go anywhere please
+      //$location.path('/');
```
Comment (mkhatib) [inline web-client/app/scripts/controllers/profileController.js]: delete the comments

File: web-client/app/scripts/controllers/profileController.js [hunk 5]
```diff
@@ -46,6 +49,7 @@ angular.module('webClientApp')
         label: angular.toJson(response.errors)
       });
       $scope.error = 'حدث خطأ في حذف المقال.';
+       $scope.inProgress = null;
```
Comment (mkhatib) [inline web-client/app/scripts/controllers/profileController.js]: Match indentation

File: web-client/app/scripts/controllers/profileController.js [hunk 6]
```diff
@@ -56,9 +60,12 @@ angular.module('webClientApp')
       $scope.inProgress = 'delete';
       if ($window.confirm('متأكد من حذف المقال؟')) {
         Article.delete({ 'articleId': article.id }, {}, deleteSuccess, deleteError);
+        var index = $scope.drafts.indexOf(article);
+        $scope.drafts.splice(index, 1);
+
```
Comment (mkhatib) [inline web-client/app/scripts/controllers/profileController.js]: drop empty line

File: web-client/app/scripts/controllers/profileController.js [hunk 7]
```diff
@@ -3,15 +3,14 @@
...
-    User.get({'userId': $routeParams.userId},
+       User.get({'userId': $routeParams.userId},
         function(resource) {
           /* jshint camelcase: false */
           $rootScope.page.title = resource.name;
           $rootScope.page.image = resource.cover_url;
           $rootScope.page.publishedTime = resource.created_at;
           $rootScope.page.description = resource.bio;
-
+
```
Comment (mkhatib) [inline web-client/app/scripts/controllers/profileController.js]: Remove spaces

File: .ruby-gemset (hunk 2 above)
Comment (ahmgeek) [inline .ruby-gemset]: yep my bad, my rvm acts weirdly on my vbox.

**Comment by mkhatib** [comment]
Awesome @ahmgeek! This looks great :smile:

**Comment by ahmgeek** [comment]
@mkhatib :open_hands:

**Comment by mkhatib** [comment]
Let me know when you address the comments I added and we'll merge this

File: web-client/app/scripts/controllers/profileController.js (hunk 3 above)
Comment (ahmgeek) [inline web-client/app/scripts/controllers/profileController.js]: just fixed it.

**Comment by ahmgeek** [comment]
good to go.

**Comment by mkhatib** [comment]
Mind doing a rebase to squash all of these in one commit?

You can do this by running `git rebase -i HEAD~3` and then just select one pick and change the other two commit messages to squash.

**Comment by ahmgeek** [comment]
Holy cow :smile: I guess I replaced the commit messages !

**Comment by mkhatib** [comment]
Well done! Thanks @ahmgeek. Sorry for the troubles, hopefully moving forward rebase won't act up.

**Comment by ahmgeek** [comment]
@mkhatib it's a gem. thanks to you.

**Pull request merged**